README.md

```diff
@@ -4,17 +4,27 @@ This directory contains the programs needed to generate and verify Sigstore root
-  * [rekor.0.pub](repository/repository/targets/artifact.pub): This is a dupe of `rekor.pub` and will be removed in the next root-signing event.
-  * [ctfe.pub](repository/repository/targets/artifact.pub): Certificate Transparency log key that is used for certificates issued by Fulcio and used to verify signed certificate timestamps (SCTs) for inclusion into the log.
-  * [artifact.pub](repository/repository/targets/artifact.pub): Key that signs Sigstore project (Cosign, Rekor, Fulcio) releases.
+* [root.json](repository/repository/root.json): This is the current `root.json`. It is signed by at least 3 out of the 5 [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders). The top-level signing keys endorsed by the root are:
+
+| Role        | TUF Key ID(s) |  Description | 
+| ----- | ------| --------- |  
+| Root    | See below | The [offline keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders).      |
+| Targets    | See below | The [offline keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders), the same as the root keyholders to minimize the number of offline keysets.       |
+| Snapshot  | `fc61191ba8a516fe386c7d6c97d918e1d241e1589729add09b122725b8c32451` | A GCP KMS [snapshotting key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L87-L97) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot`       |
+| Timestamp  | `b6710623a30c010738e64c5209d367df1c0a18cf90e6ab5292fb01680f83453d`| A GCP KMS   [timestamping key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L32-L42) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp`      |
+
+* [targets.json](repository/repository/targets.json): This is the list of trusted `targets.json` endorsed by the offline keyholders. It includes:
+
+| Target        |  Description | 
+| ----- |--------- |  
+| [fulcio_v1.crt.pem](repository/repository/targets/fulcio_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) root certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running root CA to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` | 
+| [fulcio_intermediate_v1.crt.pem](repository/repository/targets/fulcio__intermediate_v1.crt.pem)   |  This is the [Fulcio](https://github.com/sigstore/fulcio) intermediate certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running CA chain to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert` | 
```

Comment:
The URL in the curl example here is the same as the entry above

Reply:
Yeah, this returns both the root and the intermediate. I will say to match with the first/second PEM cert.

Reply:
I should have tried and learned that before commenting, thanks!